### PR TITLE
[JENKINS-51490] Caption for Abort Button

### PR DESCRIPTION
**Pull request by benpdavison**
For ticket [JENKINS-51490](https://issues.jenkins-ci.org/browse/JENKINS-51490)

I have replicated the work done to caption the Ok button for the abort button.

This is a something that I often see requested.

Please let me know if there is anything I have missed, or if i need to write a test. I noticed there was no test for the Ok button, so also followed that path.

### CODE REVIEW

**Comment by dwnusbaum** [comment]
Thanks for the PR! Is there an open Jira ticket for this issue?

> whitespace change to kick off build as node was terminated early

To get a rebuild with no code changes, you can close the PR, wait until that close is recognized at the bottom of https://ci.jenkins.io/job/Plugins/job/pipeline-input-step-plugin/indexing/events, and then reopen the PR.

**Comment by benpdavison** [comment]
Ahhh ok, yeh that might be a better way, I will revert the whitespace change. As it seems Jenkins wants to kill my PR by terminating the windows node every time.

I will have a look in Jira and see if there is a ticket.

**Comment by benpdavison** [comment]
@dwnusbaum How do I go about getting this merged in now that it is completed?

**Comment by dwnusbaum** [comment]
@benpdavison I've added some reviewers. I'll try to take a look in the next week or so. Thanks! You can test out the incremental build created by this PR [here](https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/plugins/pipeline-input-step/2.10-rc360.b5f9b64ed809/pipeline-input-step-2.10-rc360.b5f9b64ed809.hpi) if you need this functionality ASAP.

**Comment by midebell** [comment]
Submitter does not work. I was expecting it to deny non ldap users or users not included in the list, however, anyone can approve the job.. Is this a bug right now ?

File: src/test/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepTest.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 package org.jenkinsci.plugins.workflow.support.steps.input;
 
 import com.gargoylesoftware.htmlunit.ElementNotFoundException;
+import com.gargoylesoftware.htmlunit.html.DomElement;
```
Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepTest.java]: revert

**Comment by jglick** [comment]
Any particular reason this was closed?

**Comment by benpdavison** [comment]
I do not have the time to carry this on.

**Comment by mparry** [comment]
I ended up here because I was looking for this exact feature. What exactly is left to do, given that the existing changes were approved a year ago? There is maybe a stray `import` to remove, I think. Anything else?

**Comment by jglick** [comment]
@mparry nothing in particular. Needs approval from active plugin maintainer(s). What I recommend to resurrect a PR, retaining attribution of prior work, is to create a topic branch from up-to-date `master` as usual, then

```bash
git pull https://github.com/benpdavison/pipeline-input-step-plugin master
```

then add any further commits, and file your own PR. Also make sure the Jira issue is assigned to you and **In Review**.

**Pull request closed without merging**